testdata/project-v2/go.mod

```diff
@@ -1,11 +1,12 @@
 module sigs.k8s.io/kubebuilder/testdata/project-v2
 
-go 1.13
+go 1.12
```

Comment:
Note that the master branch is using go 1.13. 
In order to collab with the project, you need to use the latest master branch and have go 1.13 locally

Reply:
Thanks - updated!